code/modules/mob/living/simple_animal/friendly/mushroom.dm

```diff
@@ -23,6 +25,14 @@
 /mob/living/simple_animal/mushroom/New()
 	..()
 	harvest_time = world.time
+	var/count = 0
+	for (var/mob/living/simple_animal/mushroom in world)

```

Comment:
Not sure where that list is, but also, speed is entirely negligible here.  I adminspawned hundreds in a single operation and they all selfdeleted before becoming visible without any effect on the server. this solution works